style.css

```diff
@@ -53,45 +49,71 @@ ul {
   background: #f4f1f0;
 }
 
-/* Nav Bar */
 header {
-  font-size: 18px;
-  display: flex;
-  align-items: center;
-  justify-content: space-between;
   height: 48px;
-  text-decoration: none;
-  color: #fff;
   background: #3c3a39;
+  padding: 0 12px;
+  overflow: hidden;
 }
 
-nav {
-  padding-right: 32px;
+@media only screen and (min-width: 992px) {
```

Comment:
- [ ] I think a screen with 992 pixels is too wide for a Mobile screen size view. This makes  Mobile views show up on devices wide enough to accommodate Desktop views. it would be useful to use a recommended screen size of 768px instead of 992px